projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -152,6 +152,44 @@ public function is_task_list_enabled( $id ) {
 		return $this->load_value_from_callback( $task_list, 'is_enabled_callback', null );
 	}
 
+	/**
+	 * Check if a task list was dismissed by the user.
+	 *
+	 * @param string $id Task List id.
+	 * @return bool|null True if dismissed, false if not.
+	 */
+	public function is_task_list_dismissed( $id ) {
+		$task_list_dismissed_status = $this->get_task_list_dismissed_status();
+
+		// Return true if the task list is on the dismissed status array and its value is true.
+		return isset( $task_list_dismissed_status[ $id ] ) && $task_list_dismissed_status[ $id ];
+	}
+
+	/**
+	 * Set wether a task list is dismissed or not for a site.
+	 *
+	 * @param string $id Task List id.
+	 * @param bool   $is_dismissed True if dismissed, false if not.
+	 */
+	public function set_task_list_dismissed( $id, $is_dismissed ) {
+		$task_list_dismissed_status        = $this->get_task_list_dismissed_status();
+		$task_list_dismissed_status[ $id ] = $is_dismissed;
```

Comment:
I think we should only store cases where the `$is_dismissed` value is true so our option size doesn't get too large. I also think we need to cast the supplied `$is_dismissed` value to a boolean to make sure we're storing valid data.
```suggestion
		$is_dismissed                      = (bool) $is_dismissed;
		if ( $is_dismissed ) {
			$task_list_dismissed_status[ $id ] = $is_dismissed;
		} else {
			unset( $task_list_dismissed_status[ $id ] );
		}
```